src/test/java/com/rocketden/main/socket/RoomSocketTests.java

```diff
@@ -144,6 +149,9 @@ public void socketReceivesMessageOnHostChange() throws Exception {
         assertEquals(expected.getHost(), actual.getHost());
         assertEquals(expected.getUsers(), actual.getUsers());
 
+        // Update newUser with the created userId and updated information.
+        newUser = expected.getUsers().get(1);
```

Comment:
I've noticed that this has come up a lot where you need to iterate through the users after joining to get the updated `userId` (both in the tests and on the frontend) - do you think it'd be worth creating a new response object for these cases? For example either adding like a `currentUser` field to `RoomDto` or creating like a new `JoinRoomResponse` class that has the two fields `currentUser` and `room`

Reply:
Yeah, I think this could be a good idea! We'd want to differentiate this response object from what is sent out through the socket message though I think, because that is sent out to all users.

We can talk more about the potential design of this on the call today.

Reply:
As talked about on the call, we will look into this on a separate PR; see the [Trello card](https://trello.com/c/CjRqLj7o/161-return-current-user-info-after-a-join-room-call).